Documentation/ceph-cluster-crd.md

```diff
@@ -478,3 +480,60 @@ spec:
 ```
 
 This configuration will split replication of your volumes across unique racks in your datacenter setup.
+
+### External cluster
+
+#### Pre-requisites
+
+In order to configure an external Ceph cluster with Rook, we need to inject a couple of information in order to connect to that cluster.
+You can use the `cluster/examples/kubernetes/ceph/import-external-cluster.sh` script to achieve that.
+This script requires setting environment variables, on a Linux system you can set an environment variable like this:
```

Comment:
Well since this is needed by the script I'd say we have to.

Reply:
Instead of this general instruction, what about showing an example (after the settings are described) with the sample settings such as this?
```
export NAMESPACE=rook-ceph-external
export ROOK_EXTERNAL_FSID=3240b4aa-ddbc-42ee-98ba-4ea7b2a61514
export ROOK_EXTERNAL_ADMIN_SECRET=AQC6Ylxdja+NDBAAB7qy9MEAr4VLLq4dCIvxtg==
export ROOK_EXTERNAL_CEPH_MON_DATA=rook-ceph-mon-a=172.17.0.4:3300
```